docs/kyma/05-05-profiles.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Profiles
+type: Configuration
+---
+
+The Kyma Operator allows you to use pre-defined profiles to install or upgrade Kyma. To install Kyma using a profile, you must specify the **spec.profile** attribute in the [Installation custom resource](#custom-resource-installation) (CR).
+
+The profile is a subset of the chart's `values.yaml` defined in `profile-{name}.yaml` file at the component chart root folder.  For example, the file `profile-evaluation.yaml` defines settings for `evaluation` profile. Values from the profile override settings from `values.yaml`. A profile can override only a section, or even the whole file if required. If the profile file is not present, the Kyma Operator will use default set of values from the `values.yaml` file.
+Currently supported profiles are: 
```

Comment:
I'd move the info about profiles to the first paragraph (I think that the 2nd sentence would be the best place for it). Also, it would be great to add an explanation of these profiles - how they differ and when to use them. Please add these two sentences. If you need assistance, I am happy to help 🙂